vcr/vcr.go

```diff
@@ -305,8 +305,9 @@ func (c *vcr) Issue(template vc.VerifiableCredential) (*vc.VerifiableCredential,
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse issuer: %w", err)
 	}
-	// find did document/metadata for originating TXs
-	document, meta, err := c.docResolver.Resolve(*issuer, nil)
+	// find did document/metadata for originating TXs. ResolveMetadata is required to also check the validity of the issuer's controller.
+	now := timeFunc()
```

Comment:
is this equal to the issuance time of the VC?

Reply:
give or take a few milliseconds. We could add issuance time to the VC builder to make sure they are the same at nanosecond level?

Reply:
Would be nice (in a sense, more pure), but probably doesn't really matter much in practice.